mllib/src/main/scala/org/apache/spark/mllib/evaluation/binary/BinaryClassificationMetricComputers.scala

```diff
@@ -24,6 +24,19 @@ private[evaluation] trait BinaryClassificationMetricComputer extends Serializabl
   def apply(c: BinaryConfusionMatrix): Double
 }
 
+/** Accuracy. Defined as 1.0 where there are zero no positive and negative examples. */
```

Comment:
"... are zero no positive and negative ..." should probably be "... are no positive or negative ..."?